README.md

```diff
@@ -46,6 +46,21 @@ We pass any provided options into the `imagemin` itself:
   ];
 }
 ```
+### WebP Support
+```js
+import imageminWebp from 'imagemin-webp'
+
+module.exports = {
+  modules: [
+    ['nuxt-imagemin', {
+      plugins: [
+        imageminWebp({quality: 50})
+      ]
+    }]
+  ]
+}
+```
+> credits: [@ricardogobbosouza](https://github.com/wemake-services/nuxt-imagemin/issues/67#issuecomment-463707466)
```

Comment:
Sorry, but posting this in a README is not a good thing. Not because I do not want to give you a credit for that, but just because this information is not valuable for the reader.

README is an entry point to the project. It must contain only valuable parts to the reader, to convince to use this library or to answer specific questions. So, let's turn it to  the question. You can do something like:

```md
If you [wonder](https://github.com/wemake-services/nuxt-imagemin/issues/67) how to add custom plugins ...
```

In this form it is perfectly acceptable.